app/questionnaire/router.py

```diff
@@ -267,7 +288,7 @@ def _get_last_complete_section_key(self) -> Tuple[str, Optional[str]]:
             if self._progress_store.is_section_complete(section_id, list_item_id):
                 return section_id, list_item_id
 
```

Comment:
We should replace the type for this function from `Tuple` to `tuple`. We should use built-in types over types from mypy.

Reply:
Done, thanks!